build-system/server/app.js

```diff
@@ -411,6 +411,35 @@ app.use('/form/autocomplete/error', (req, res) => {
   res(500);
 });
 
+const mentionNames = [
```

Comment:
`app.js` is rather large already.

We should probably do a larger refactoring effort separately, but could you put this test set in a separate file? That way you can also name it for context.

Reply:
The `build-system/server/routes folder` has per-component routes. We could make a file for `form` routes if appropriate.

Reply:
Oooh, nice suggestion Cathy! Filed #25374

Reply:
I think that'd be a separate task, you can create the test set file and add the route later.